src/extension.ts

```diff
@@ -14,6 +14,23 @@ import {
   TextDocumentIdentifier,
 } from "vscode-languageclient";
 import { getCoursierExecutable } from "./coursier/coursier";
+import { TextDecoder } from "util";
+
+type Organization = string;
+type Artifact = string;
+type Version = string;
+type MavenCoordinate = `${Organization}:${Artifact}:${Version}`;
+
+interface SmithyBuild {
+  imports?: Array<string>;
+  mavenRepositories?: Array<string>; //deprecated
+  mavenDependencies?: Array<string>; //deprecated
+  maven?: {
+    dependencies?: Array<string>;
+    repositories?: Array<{ url: string }>;
+  };
+  languageServer?: MavenCoordinate;
```

Comment:
Is it needed to read all of these fields? As far as I can see we're only interested in `languageServer`

Reply:
I guess the content is read anyway, I'm just being explicit about the content, but yeah I could remove most have it and use `PartialSmithyBuild instead

Reply:
yeah, it's gonna be the same either way as you're not really validating the file contents (`as` gets erased in the output javascript)

Reply:
if it were me, I'd keep it, I can add a not but I would keep it.

but if you really want it remove and I can, also wdyt @Baccata ?

Reply:
I think Jakub is right, but I'm merging anyway because time differences